README.md

```diff
@@ -59,18 +59,19 @@ To get a local development version of `0x-api` running:
-| `TAKER_FEE_ASSET_DATA`  | `0x`                                                            | The taker fee token asset data for created 0x orders.                                                                                                                                  |
-| `MAKER_FEE_UNIT_AMOUNT` | `0`                                                             | The flat maker fee amount you'd like to receive for filled orders hosted by you.                                                                                                       |
-| `TAKER_FEE_UNIT_AMOUNT` | `0`                                                             | The flat taker fee amount you'd like to receive for filled orders hosted by you.                                                                                                       |
-| `WHITELIST_ALL_TOKENS`  | `false`                                                         | A boolean determining whether all tokens should be allowed to be posted.                                                                                                               |
+| Environment Variable    | Default                                                                                      | Description                                                                                                                                                                            |
+| ----------------------- | -------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `CHAIN_ID`              | Required. No default.                                                                        | The chain id you'd like your API to run on (e.g: `1` -> mainnet, `42` -> Kovan, `3` -> Ropsten, `1337` -> Ganache). Defaults to `42` in the API, but required for `docker-compose up`. |
+| `ETHEREUM_RPC_URL`      | Required. No default.                                                                        | The URL used to issue JSON RPC requests. Use `http://ganache:8545` to use the local ganache instance.                                                                                  |
+| `MESH_WEBSOCKET_URI`    | Required. Default for dev: `ws://localhost:60557`                                            | The URL pointing to the 0x Mesh node. A default node is spun up in `docker-compose up`                                                                                                 |
+| `MESH_HTTP_URI`         | Optional. Used when syncing the orderbook; defaults to websocket connection if not provided. | The URL pointing to the Mesh node's HTTP JSON-RPC endpoint. A default node is spun up in `docker-compose up`                                                                           |
```

Comment:
Nit: I would just say "Optional. No default." and then put the description part in the description column.